### ISSUE TITLE
Environment -> Tag changes are not saved

### ISSUE THREAD

**Issue by lordzsolt** [getinsomnia/insomnia#6466]
### Expected Behavior

- All changes performed to the Tag are saved

### Actual Behavior

- Request and Trigger Behavior is not saved

### Reproduction Steps

1. Manage Environments
2. Create a Tag
	Ex: 	`"LAST_RECIPE_ID": "{% response 'body', 'req_b8dafc71cbb548e188e713b5a093c5e6', 'b64::JC5pZA==::46b', 'always', 60 %}"`
![Screenshot 2023-09-08 at 14 57 20](https://github.com/Kong/insomnia/assets/8976891/eb050559-160c-42ea-b7ff-56547d50c4eb)

3. Close Environments
4. Manage Environments
5. Click the Tag to bring up the Edit Window
6. Change `Trigger Behavior` -> Never
7. Press `Done`
8. Close Environments
9. Manage Environments
10. Click the Tag to bring up the Edit Window
11. Notice the `Trigger Behavior` is reverted back to `Always`

### Is there an existing issue for this?

- [X] I have searched the [issue tracker](https://www.github.com/Kong/insomnia/issues) for this problem.

### Additional Information

It seems like if I highlight the tag after Step 7. and copy it, the modified tag appears in the clipboard and also the bug doesn't happen.

Not sure if it's some weird issue with the file not being correctly synched back to the disk?

Imho, it would be good to have a `Save` button in the Environments, that would force a write-out to disk.

### Insomnia Version

2023.5.8

### What operating system are you using?

macOS

### Operating System Version

13.2.1 (22D68)

### Installation method

Homebrew

**Comment by lordzsolt** [comment]
I don't think this is related to the latest release, I think it was broken in the previous one as well.

**Comment by lordzsolt** [comment]
Looks like it's the same issue as #5612

**Comment by corebreaker** [comment]
It seems to be liked to the issue #6372

**Comment by corebreaker** [comment]
It seems solved since Insomnia 9.3 by #6943 .

I think this issue can be closed.
